charts/sauron/templates/sauron-commands.yaml

```diff
@@ -146,6 +146,146 @@ data:
     image_auto_update valkyrie {{ .Values.deployments.valkyrie.major }} {{ .Values.deployments.valkyrie.minor }}
     {{- end }}
 
+  deployment_update_script.sh: |
+    #!/bin/bash
+    
+    HOME=/home
+    
+    re="^https:\/\/github.com\/(.+)\/(.+).git*$"
+    
+    if [[ "$GIT_HTTPS_CLONE_URL" =~ $re ]]; then
+      USER=${BASH_REMATCH[1]}
+      echo "User: $USER"
+      printf "\n\n"
+    
+      REPO=${BASH_REMATCH[2]}
+      echo "Repo: $REPO"
+      printf "\n\n"
+    fi
+    
+    echo "Target Branch: $TARGET_BRANCH"
+    printf "\n\n"
+    TARGET_DEPLOYMENT_SHA=$(curl \
+      -H "Accept: application/vnd.github+json" \
+      -H "Authorization: Bearer $SAURON_PROXY_PAT" \
```

Comment:
Create external GitHub account for usage with the pat using urbanos_devs@accenture.com